### ISSUE TITLE
Release SteamGuard OTP support

### ISSUE THREAD

**Issue by JamesTheAwesomeDude** [freeotp/freeotp-android#250]
#105, which was merged into master 3 years ago, is still not available in the version offered on the Play Store.

Could a release be issued with, if nothing else, that patch?

**Comment by BlkPingu** [comment]
Is this available and has a tutorial how to add steam guard to freeOTP? I'm having the problem right now that I don't want to use steam own Steam Guard app and I can't find info how to add their stuff to freeOTP on iOS

**Comment by JamesTheAwesomeDude** [comment]
@BlkPingu It's not available; sadly, this project is dead.

[FreeOTP+](https://github.com/helloworld1/FreeOTPPlus), which is available for Android, has this functionality: https://github.com/helloworld1/FreeOTPPlus/pull/113

To extract the OTP secret for use with a non-Steam app, though, you'll either have to root your phone, or use a third-party program such as https://github.com/ValvePython/steam

**Comment by quinndiggity** [comment]
@BlkPingu, the process is:

- you *must* have a phone number added to your Steam account (note: Steam are asshats around whatever they assume is a VoIP number)
- leverage a utility such as [steamctl](https://github.com/ValvePython/steamctl) to access the secret key for your account (which also allows the proprietary app to work in parallel; FreeOTP, steamctl, and the Steam app will all work concurrently) + generate a QR code for easily importing the account to FreeOTP

On the `steamctl` side, you'd be looking at the following process:

```
# setup:
pip3 install steamctl
steamctl authenticator add "$STEAM_USERNAME"

# qrcode:
steamctl authenticator qrcode

# alternative qrcode approach:
apt install jq
pip3 install "qrcode[pil]"
cat "$HOME/.local/share/steamctl/authenticator/${STEAM_USERNAME}.json" | jq -r .uri | qr --ascii
```

Another note, Steam does make their app available (which doesn't depend upon gplay) [here](https://store.steampowered.com/mobile#mobile_section) (`Android users without access to Google Play can download...`)
![image](https://user-images.githubusercontent.com/10092236/223962198-795fe63a-66d5-4ea9-b13f-9796b9d7638f.png) - going through the setup once can be useful to ensure you have Steam's requirements around an associated phone number with your account, etc, fulfilled.
